2023/barber-pole-1/hungarian/sentence_translations.json

```
@@ -704,7 +704,7 @@
  {
   "input": "The first question requires understanding circularly polarized light, since the key is that sucrose is a chiral molecule, which is to say there's a handedness to it.",
   "model": "nmt",
-  "translatedText": "Az első kérdés megköveteli a cirkulárisan polarizált fény megértését, mivel a kulcs az, hogy a szacharóz egy királis molekula, ami azt jelenti, hogy van egy kézügyessége.",
+  "translatedText": "Az első kérdés megköveteli a körkörösen polarizált fény megértését, mivel a kulcs az, hogy a szacharóz egy királis molekula, ami azt jelenti, hogy a tükörképi párjukkal fedésbe hozhatók.",
```

```suggestion
  "translatedText": "Az első kérdés megköveteli a körkörösen polarizált fény megértését, mivel a kulcs az, hogy a szacharóz egy királis molekula, ami azt jelenti, hogy a tükörképi párjukkal nem hozhatók fedésbe.",
```
Important difference in meaning! Chiral means that if there is a molecule and its pair which is the exact mirror image, then these two are different from eachother. "Handedness" refers to this, that it's like a right hand and a left hand, which are mirror images of each other.